.github/COMMIT_CONVENTION.md

```diff
@@ -40,12 +40,21 @@ Must be one of the following:
 
 ### Scope:
 
-The optional scope allows to specify the place of the change. For instance, if the commit affects a specific component, use the component's name as scope:
+The optional scope allows to specify the place of the change.
+For instance, if the commit affects a specific component, use the component's name as scope:
+
+Note: use lowercase and kebab-case syntax for the scope, that means all in lowercase and separate the words with dash.
 
 ```
 feat(select-menu): add leftIcon prop
 ```
 
+In case of multiple scopes, separate them with comma.
+
+```
+fix(combobox, combobox-with-popover): fix keyboard navigation 
```

Comment:
> Do you think this will be an issue with messages like this? It's especially a problem with recipe components which usually have longer more descriptive names.

Yeah, 72 chars are not enough in cases where the scope is a recipe and the summary is a bit large. So I increased the limit in commitlint to 120. https://github.com/dialpad/dialtone-vue/pull/537/commits/e5482d3cd40b5560c310cc6c2cd8148773622161

> What happens if they pass in a component name that doesn't exist in our system? Does this matter for our release notes?

No. There's no validation for the scope of the commit message, just formatting the scope for the release notes from [kebab-case to Title Case](https://github.com/dialpad/conventional-changelog-angular/commit/19492e83fbef2a160532978bd596d7ced486a239).

Reply:
> Yeah, 72 chars are not enough in cases where the scope is a recipe and the summary is a bit large. So I increased the limit in commitlint to 120. [e5482d3](https://github.com/dialpad/dialtone-vue/commit/e5482d3cd40b5560c310cc6c2cd8148773622161)

oh nice, yeah that's better